applications/MeshMovingApplication/custom_strategies/strategies/laplacian_meshmoving_strategy.h

```diff
@@ -192,9 +192,6 @@ class LaplacianMeshMovingStrategy
       mstrategy_z->Solve();
     }
 
-    MoveMeshUtilities::MoveMesh(
```

Comment:
@adityaghantasala this is what I mean, you should do the same in the Trilinos Stragegies

Reply:
Ok !